Examples/MagicWeatherSwiftUI/Magic Weather SwiftUI.xcodeproj/project.pbxproj

```diff
@@ -10,9 +10,9 @@
 		55413BE725B735AF00ECCA5A /* UserView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 55413BE625B735AF00ECCA5A /* UserView.swift */; };
 		55413BEC25B735C500ECCA5A /* WeatherView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 55413BEB25B735C500ECCA5A /* WeatherView.swift */; };
 		55413BF125B73A6F00ECCA5A /* PaywallView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 55413BF025B73A6F00ECCA5A /* PaywallView.swift */; };
-		55413C0025B778E200ECCA5A /* WeatherViewModel.swift in Sources */ = {isa = PBXBuildFile; fileRef = 55413BFF25B778E200ECCA5A /* WeatherViewModel.swift */; };
-		55413C0425B778F700ECCA5A /* UserViewModel.swift in Sources */ = {isa = PBXBuildFile; fileRef = 55413C0325B778F700ECCA5A /* UserViewModel.swift */; };
-		55413C0D25C07B7900ECCA5A /* Extensions.swift in Sources */ = {isa = PBXBuildFile; fileRef = 55413C0C25C07B7900ECCA5A /* Extensions.swift */; };
+		554.4.0-SNAPSHOT025B778E200ECCA5A /* WeatherViewModel.swift in Sources */ = {isa = PBXBuildFile; fileRef = 55413BFF25B778E200ECCA5A /* WeatherViewModel.swift */; };
```

Comment:
This is wrong

Reply:
This logic is wrong somehow: https://github.com/RevenueCat/purchases-ios/blob/main/fastlane/Fastfile#L609-L611

Reply:
It's happened before and yeah I have no idea.

Reply:
Actually I have a theory.

Reply:
Okay I know why, fixing

Reply:
Fixed #1536 and updated this PR